### PR TITLE
Updates for decrypt tool

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

To add the encrypt/decrypt tool to the documentation page along with a description of how to run it.

## Changes

Added three new topics, developer-tools, where two of the topics are just references to the main topic in the shared folder. Updated 2x downloads.rst and 2 x index.rst.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

### CODE REVIEW

File: source/shared/net/references/developer-tools.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+.. contents::
+   :local:
+   :backlinks: none
+
+
+Decrypt encrypted output
+========================
+
+Some Concordium tools, such as the Mobile Wallet, sometimes produce encrypted output. These files can be decrypted and inspected using the **utils** tool . You can download the tool from **reference here**.
```
Comment (abizjak) [inline source/shared/net/references/developer-tools.rst:15]: Is the **reference here** intended?

File: source/shared/net/references/developer-tools.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+To encrypt, enter:
+
+``utils encrypt --in input.blah --out output.json``
```
Comment (abizjak) [inline source/shared/net/references/developer-tools.rst:19]: Maybe you could choose a better name than `input.blah` that I chose, perhaps `input.example`.

File: source/shared/net/references/developer-tools.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+The code example above results in the following interaction:
+
+| ``Enter password to encrypt with: ....``
+| ``Re-enter password: ....``
+| ``Writing output to output.json``
+
+To decrypt, enter:
+
+``utils decrypt --in output.json --out decrypted.blah``
```
Comment (abizjak) [inline source/shared/net/references/developer-tools.rst:31]: Same here.

File: source/shared/net/references/developer-tools.rst [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+Enter password to decrypt with, writing output to **decrypted.blah**.
```
Comment (abizjak) [inline source/shared/net/references/developer-tools.rst:33]: And then of course here.

File: source/shared/net/references/developer-tools.rst (hunk 1 above)
Comment (dg-concordium) [reply source/shared/net/references/developer-tools.rst]: No, just me forgetting to go back and change the placeholder I put in.

**Review by LotteKh — APPROVED** [review]
I'm unsure whether Developer Tools is the right heading? We have other tools that are developer-oriented, so I don't know if this could be confusing?

**Comment by abizjak** [comment]
@LotteKh would Auxiliary tools be better?

File: source/shared/net/references/developer-tools.rst [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+
+Decrypt encrypted output
+========================
+
+Some Concordium tools, such as the Mobile Wallet, sometimes produce encrypted output. These files can be decrypted and inspected using the **utils** tool . You can download the tool from :ref:`Installation downloads <downloads>`.
+
+To encrypt, enter:
+
+``utils encrypt --in input.example --out output.json``
```
Comment (abizjak) [inline source/shared/net/references/developer-tools.rst:19]: I think my previous comment was misleading. To be consistent with the decrypt line this should be
```
utils encrypt --in input.json --out output.example
```

File: source/shared/net/references/developer-tools.rst [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+where ``--in ...`` is the file to encrypt, and ``--out`` is the file where encrypted data will be written.
+
+The code example above results in the following interaction:
+
+| ``Enter password to encrypt with: ....``
+| ``Re-enter password: ....``
+| ``Writing output to output.json``
```
Comment (abizjak) [inline source/shared/net/references/developer-tools.rst:27]: And then the corresponding change here.

**Review by abizjak — APPROVED** [review]
If you can fix the naming of the file in the example  I think this is good to go.

**Pull request merged**